src/pymoca/ast.py

```diff
@@ -24,7 +25,6 @@ class FoundElementaryClassError(Exception):
 
 
 class Visibility(Enum):
-    PRIVATE = 0, "private"
```

Comment:
Related to this, there are underlying questions that come to mind with respect to spec compliance. How much do we want to check and enforce? Do we assume that the user of Pymoca has checked the Modelica code in some other tool before feeding it into Pymoca and just not do the checks? This would make Pymoca simpler and faster. Are there cases where if we don't do the check we can actually interpret correct Modelica code incorrectly? I don't know.

Reply:
We talk error checking more recently in https://github.com/pymoca/pymoca/pull/307#issuecomment-2211148462, and we have more recently talked about visibility also, so I'm resolving this stale comment thread.